Plugins/BenchmarkBoilerplateGenerator/BenchmarkBoilerplateGenerator.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) 2023 Ordo One AB.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+
+import ArgumentParser
+import SystemPackage
+
+@main
+struct Benchmark: AsyncParsableCommand {
+    @Option(name: .long, help: "Name of the target")
+    var target: String
+
+    @Option(name: .long, help: "Output file path")
+    var output: String
+
+    mutating func run() async throws {
+        let outputPath = FilePath(output) // package
+        var boilerplate = """
```

Comment:
Can't be `let`?

Reply:
Nope, `boilerplate.withUTF8` requires a mutable var.